test/units/testsuite-17.11.sh

```diff
@@ -132,7 +132,7 @@ cp "${workdir}/default_output_1_fail" "${exo}"
 assert_1 "${rules}"
 
 {
-    printf 'RUN+="/bin/true"%8175s\\\n' ' '
+    printf 'RUN+="/bin/true",%8174s\\\n' ' '
     printf 'RUN+="/bin/false"%8174s\\\n' ' '
```

Comment:
I was a bit confused by this. We write two lines like `RUN+="..."                \`. The second one is followed by an empty line, so the continuation symbol just means that we append some whitespace to the logical line. So adding a comma just in the first `RUN` line seems correct.